packages/remix/src/generators/library/library.impl.ts

```diff
@@ -1,13 +1,9 @@
-import type { Tree } from '@nx/devkit';
-import { formatFiles, GeneratorCallback, runTasksInSerial } from '@nx/devkit';
-import { Linter } from '@nx/linter';
-import { libraryGenerator } from '@nx/react/src/generators/library/library';
-import {
-  addTsconfigEntryPoints,
-  normalizeOptions,
-  updateBuildableConfig,
-} from './lib';
-import type { NxRemixGeneratorSchema } from './schema';
+import type {Tree} from '@nx/devkit';
+import {formatFiles, GeneratorCallback, runTasksInSerial} from '@nx/devkit';
+import {Linter} from '@nx/linter';
+import {libraryGenerator} from '@nx/react/src/generators/library/library';
```

Comment:
i'm not sure if it can be imported without a deep import but i can find out, and if it isn't i can take a look at nx/react and see if it can be set up that we don't need a deep import

Reply:
it seems it can!